pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -279,6 +302,36 @@ func (cm *containerManagerImpl) NewPodContainerManager() PodContainerManager {
 	}
 }
 
+func (cm *containerManagerImpl) InternalContainerLifecycle() InternalContainerLifecycle {
```

Comment:
@ConnorDoyle -- explained why this is off ContainerManager and not PodContainerManager in person review, and this seemed fine for me... major concern was that its not off the top-level kubelet.